lib/features/family/features/reflect/presentation/pages/reveal_secret_word.dart

```diff
@@ -107,13 +109,24 @@ class _RevealSecretWordScreenState extends State<RevealSecretWordScreen> {
             FunButton(
               isDisabled: !_isSecretWordVisible,
               onTap: () {
-                final sidekick = _cubit.getSidekick();
-                Navigator.of(context).pushReplacement(
-                  PassThePhone.toSidekick(
-                    sidekick,
-                    toRules: true,
-                  ).toRoute(context),
-                );
+                final isFirstRound = _cubit.isFirstRound();
+                if (isFirstRound) {
+                  final sidekick = _cubit.getSidekick();
+                  Navigator.of(context).pushReplacement(
+                    PassThePhone.toSidekick(
+                      sidekick,
+                      toRules: true,
+                    ).toRoute(context),
+                  );
+                } else {
+                  final reporters = getIt<InterviewCubit>().getReporters();
+                  Navigator.of(context).pushReplacement(
+                    PassThePhone.toReporters(
+                      reporters,
+                      skipRules: true,
```

Comment:
confusing that there is both 'toRules' and 'skipRules' for PassThePhone constructors

Reply:
True but they are not for the same constructor! One is for the sidekick the other one for the reporters. If it were the same I would not make a double boolean but an enum/ different constructors instead.